style.css

```diff
@@ -8,7 +8,7 @@
 
 body {
     font-family: 'Lato', 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
-    background-color:#09437e;
+    background-color:#084274;

```

Comment:
I don't like this color, can you change it?

Reply:
Sure! I have changed it.